numba/cpython/builtins.py

```diff
@@ -604,6 +613,15 @@ def max_impl(indval1, indval2):
         return max_impl
 
 
+@overload(max)
+def boolval_max(val1, val2):
+    if isinstance(val1, types.scalars.Boolean) and \
+       isinstance(val2, types.scalars.Boolean):
```

Comment:
```suggestion
    if isinstance(val1, types.Boolean) and \
       isinstance(val2, types.Boolean):
```
suggest using the top level re-export for consistency.